### PR TITLE
docs: Update getting started with VsCode

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?
"Getting started with VSCode" is not completed

## What is the new behavior?
Update "Getting Started with VsCode" 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by ajpinedam** [comment]
If you wanna see the final document, it can be read here:

https://gist.github.com/ajpinedam/57e0f39cc60634324e847eed346c147f

File: doc/articles/get-started-vscode.md [hunk 1]
```diff
@@ -1,74 +1,118 @@
...
+
+#### Create the project
+
+In the terminal type the following to create a new project:
+
 ```bash
 dotnet new unoapp -o MyApp -ios=false -android=false -macos=false -uwp=false --vscodeWasm
 ```
 
+> `MyApp` is the name you want to give it to your project.
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-vscode.md:35]:
```diff
- is the name you want to give it to your project.
+ is the name you want to give to your project.
```
@davidjohnoliver not sure if I'm not mistaking here ;) thanks in advance for the feedback !

**Review by agneszitte-nventive — APPROVED** [review]

File: doc/articles/get-started-vscode.md [hunk 2]
```diff
@@ -1,74 +1,118 @@
...
+    > Use `dotnet --version` from the terminal to get the version installed.
+
 ### Create an Uno Platform project
 
-1. Launch Code, then in the terminal type the following to install the Uno Platform templates:
+#### Install Uno Platform Template
+
+Launch Visual Studio Code and open a new terminal:
+
+In the terminal type the following to install the Uno Platform templates:
```
Comment (davidjohnoliver) [inline doc/articles/get-started-vscode.md:21]:
```suggestion
In the terminal, type the following to install the Uno Platform templates:
```

File: doc/articles/get-started-vscode.md [hunk 3]
```diff
@@ -1,74 +1,118 @@
...
+* [**.Net Core SDK**](https://dotnet.microsoft.com/download) (**version 3.1.8** or later)
+
+    > Use `dotnet --version` from the terminal to get the version installed.
+
 ### Create an Uno Platform project
 
-1. Launch Code, then in the terminal type the following to install the Uno Platform templates:
+#### Install Uno Platform Template
+
+Launch Visual Studio Code and open a new terminal:
```
Comment (davidjohnoliver) [inline doc/articles/get-started-vscode.md:19]:
```suggestion
Launch Visual Studio Code and open a new terminal.
```

File: doc/articles/get-started-vscode.md [hunk 4]
```diff
@@ -1,74 +1,118 @@
...
+In the terminal type the following to install the Uno Platform templates:
+
 ```bash
 dotnet new -i Uno.ProjectTemplates.Dotnet
 ```
-2. In the terminal type the following to create a new project:
+
+#### Create the project
+
+In the terminal type the following to create a new project:
```
Comment (davidjohnoliver) [inline doc/articles/get-started-vscode.md:29]:
```suggestion
In the terminal, type the following to create a new project:
```

File: doc/articles/get-started-vscode.md (hunk 1 above)
Comment (davidjohnoliver) [reply doc/articles/get-started-vscode.md:35]: yep @agneszitte-nventive you're correct :)

**Review by davidjohnoliver — APPROVED** [review]

File: doc/articles/get-started-vscode.md [hunk 5]
```diff
@@ -1,74 +1,118 @@
...
-This guide will walk you through the set-up process for building WebAssembly apps with Uno, under Windows, Linux or macOS.
+This guide will walk you through the set-up process for building WebAssembly apps with Uno under Windows, Linux, or macOS.
 
 ### Prerequisites
+
 * [**Visual Studio Code**](https://code.visualstudio.com/)
 
+* [**Mono**](https://www.mono-project.com/download/stable/)
+
+* [**.Net Core SDK**](https://dotnet.microsoft.com/download) (**version 3.1.8** or later)
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:11]:
```suggestion
* [**.Net Core SDK**](https://dotnet.microsoft.com/download) (**version 3.1.8 (SDK 3.1.402)** or later)
```

File: doc/articles/get-started-vscode.md [hunk 6]
```diff
@@ -1,74 +1,118 @@
...
-                    FontSize="30" /> 
-        <Button Content="click" 
-                Click="OnClick" /> 
+    <StackPanel>
+        <TextBlock x:Name="txt"
+                    Text="Hello, world!"
+                    Margin="20"
+                    FontSize="30" />
+        <Button Content="click"
+                Click="OnClick" />
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:70]:
```suggestion
                Click="{x:Bind OnClick}" />
```

File: doc/articles/get-started-vscode.md [hunk 7]
```diff
@@ -1,74 +1,118 @@
...
     ```
+
 2. In your `MainPage.xaml.cs`, add the following method:
+
     ```csharp
-    public void OnClick(object sender, object args) 
-    { 
-        var dt = DateTime.Now.ToString(); 
-        txt.Text = dt; 
+    public void OnClick(object sender, object args)
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:77]:
```suggestion
    private void OnClick()
```

File: doc/articles/get-started-vscode.md [hunk 8]
```diff
@@ -1,74 +1,118 @@
...
+If you already have an Uno application, you can add some missing support files for VS Code to recognize your project.
 
 Here's how to do this:
+
 1. Use the same command line above to create a project with the same name as your current project, in a different location.
 2. Once created, copy the generated `.vscode` folder next to your `.sln` file
 3. Update the `Uno.UI` package version to the latest stable version
-4. Update the `Uno.Wasm.Bootstrap` package to 1.2.0 or later version
-5. Add a reference to `Uno.Wasm.Bootstrap.DevServer` version 1.2 or later.
+4. Update the `Uno.Wasm.Bootstrap` package to 1.4.0 or later version
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:105]:
```suggestion
4. Update the `Uno.Wasm.Bootstrap` package to 1.3.4 or later version
```

File: doc/articles/get-started-vscode.md [hunk 9]
```diff
@@ -1,74 +1,118 @@
...
+5. Add a reference to `Uno.Wasm.Bootstrap.DevServer` version 1.4.0 or later.
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:106]:
```suggestion
5. Add a reference to `Uno.Wasm.Bootstrap.DevServer` version 1.3.4 or later.
```

**Review by jeromelaban — APPROVED** [review]

File: doc/articles/get-started-vscode.md (hunk 8 above)
Comment (jeromelaban) [reply doc/articles/get-started-vscode.md:105]: Bootstrapper 1.4 is not stable yet

File: doc/articles/get-started-vscode.md [hunk 10]
```diff
@@ -1,74 +1,118 @@
...
+
+#### Create the project
+
+In the terminal, type the following to create a new project:
+
 ```bash
 dotnet new unoapp -o MyApp -ios=false -android=false -macos=false -uwp=false --vscodeWasm
 ```
 
+> `MyApp` is the name you want to give it to your project.
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-vscode.md:35]:
```suggestion
> `MyApp` is the name you want to give to your project.
```

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**